### PR TITLE
fix: don't include partial scss files in bundle

### PR DESCRIPTION
**Pull request by vakrilov**
## PR Checklist

- [x] The PR title follows our guidelines: https://github.com/NativeScript/NativeScript/blob/master/CONTRIBUTING.md#commit-messages.
- [x] There is an issue for the bug/feature this PR is for. To avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it.
- [x] You have signed the [CLA].
- [x] All existing tests are passing: https://github.com/NativeScript/nativescript-dev-webpack/blob/master/CONTRIBUTING.md#testing-locally-by-running-e2e-tests
- [x] Tests for the changes are included.

## What is the current behavior?
Partial `.scss` are included in the bundle by `BundleConfigLoader`.

## What is the new behavior?
Partial `.scss` are **not** included in the bundle by `BundleConfigLoader`.

### CODE REVIEW

File: bundle-config-loader.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { getOptions } from "loader-utils";
 import * as escapeRegExp from "escape-string-regexp";
 
 // Matches all source, markup and style files that are not in App_Resources
-const defaultMatch = "(?<!App_Resources.*)\.(xml|css|js|(?<!d\.)ts|scss)$";
+const defaultMatch = "(?<!App_Resources.*)\\.(xml|css|js|(?<!\\.d\\.)ts|(?<!_[\\w-]*\\.)scss)$";
```
Comment (rosen-vladimirov) [inline bundle-config-loader.ts]: This RegExp will not match folders containing App_Resources in their name:
App_Resources123, MyApp_Resources, etc. Is this expected?
If not, you can change the RegExp to:
```JavaScript
const defaultMatch = "(?<!\\bApp_Resources\\b.*)\\.(xml|css|js|(?<!\\.d\\.)ts|(?<!_[\\w-]*\\.)scss)$";
```

File: package.json [hunk 2]
```diff
@@ -33,7 +33,7 @@
     "preuninstall": "node preuninstall.js",
     "postpack": "rm -rf node_modules",
     "prepare": "npm run tsc && npm run jasmine",
-    "test": "npm run prepare && npm run jasmine",
+    "test": "npm run prepare",
```
Comment (rosen-vladimirov) [inline package.json]: What's the purpose of this change?

File: bundle-config-loader.ts (hunk 1 above)
Comment (rosen-vladimirov) [reply bundle-config-loader.ts]: If I understand this RegExp correctly, we do not allow having `_` in the name of the scss file, i.e.
`app_component.scss` will not match. Am I right? Is this the intended behavior?

Comment (rosen-vladimirov) [reply bundle-config-loader.ts]: If we want to allow having `_` in the names of the `scss` files, we can add `\\b` in the negative lookbehind statement. Combining the RegExp with the suggestion from my previous comment, it would look like:
```JavaScript
const defaultMatch = "(?<!\\bApp_Resources\\b.*)\\.(xml|css|js|(?<!\\.d\\.)ts|(?<!\\b_[\\w-]*\\.)scss)$"
```

Comment (rosen-vladimirov) [reply bundle-config-loader.ts]: I've tried to test the regexp (the suggestion from my comment) and I've noticed the following file still matches the RegExp `_random'.scss"`. But... I do not expect anyone to have such file.

File: package.json (hunk 2 above)
Comment (vakrilov) [reply package.json]: prepare already runs tests too, no need to `npm run jasmine` twice

File: bundle-config-loader.ts (hunk 1 above)
Comment (vakrilov) [reply bundle-config-loader.ts]: Yup. Currently using `[\w-]*` to match the file name. 
Also `_patial;drop-table-users.scss` is a potential SQL injection vulnerability 🙊

**Comment by vakrilov** [comment]
test branch_ngapps#release ngapps#release
